tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -121,6 +121,9 @@ export class WebGPUBackend extends KernelBackend {
   private uploadWaitMs = 0;
   private downloadWaitMs = 0;
   private cpuBackend: KernelBackend;
+  // TODO: Remove once WebGPU timestamp query returns timestamps instead of
+  // ticks number
+  private GPU_FREQUENCY = 12000048;
```

Comment:
Remove it since it's only for windows. I think it's already time unit for mac. Let's keep it as it is and do extra processing in user side.

Reply:
Done